index.html

```diff
@@ -269,47 +318,103 @@ <h4>SvgRenderingTemplate</h4>
             <tr>
               <td>name</td>
               <td>
-A name that can be used by a display to express the type of rendering that
-is associated with the render method. This property might be used in a
+An OPTIONAL human-readable [=string=] that can be displayed to provide a hint to
+the type of rendering that will be performed. This property might be used in a
 graphical interface that enables an individual to select between multiple
 presentation modes.
               </td>
             </tr>
             <tr>
-              <td>css3MediaQuery</td>
+              <td>description</td>
               <td>
-A CSS Level 3 Media Query that can be used to specify desired characteristics
-in the display device. If there are multiple render methods, the best match
-to the CSS Level 3 Media Query will be used by default. It is expected that
-an individual could override this preference through a manual process.
+An OPTIONAL human-readable [=string=] that provides a more involved description
+than `name` of when the particular rendering might be useful.
+              </td>
+            </tr>
+            <tr>
+              <td>disclosedProperty</td>
+              <td>
+An OPTIONAL [=list=] of [=string=] values that each conform to the
+[[[RFC6901]]] syntax that specifies which properties from the [=verifiable
+credential=] are exposed when using this specific render method. If
+`disclosedProperty` is not provided, the entire [=verifiable credential=] is
+presumed to be shared when the render method is used.
+              </td>
+            </tr>
+            <tr>
+              <td>template</td>
+              <td>
+An OPTIONAL [=string=] or [=ordered map|map=] that provides the template that
+will be used to perform the rendering. If the value is a [=string=], it MUST be
+a URL. If the value is a [=ordered map|map=], it MUST conform to the following
+rules:
+                <table class="simple">
+                  <thead>
+                    <tr>
+                      <th style="white-space: nowrap">Property</th>
+                      <th>Description</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <tr>
+                      <td>id</td>
+                      <td>
+An REQUIRED [=string=] that follows the [[[URL]]] and, when fetched,
+dereferences to a template such as an SVG or PDF file.
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>mediaType</td>
+                      <td>
+A RECOMMENDED [=string=] that identifies the media type for the `id` value
+as specified in [[[RFC6838]]].
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>digestMultibase</td>
+                      <td>
+An OPTIONAL multibase-encoded Multihash of the template file. The multibase
+value MUST be `z` and the multihash value MUST be SHA-2 with 256-bits of output
+(`0x12`).
```

Comment:
I think this should be base64url-nopad prefix `u` if only one option is allowed (as a more commonly available / used base encoding for hashes).

Reply:
I agree with @dlongley here, we should go with `u` by default.

Reply:
That makes sense assuming that we are only using one prefix. Since this property is named `digestMultibase` and not `digestMultibaseBase64Url`, we should allow any valid multibase value here.

We could then remove this statement:

```
The multibase
value MUST be `z` and the multihash value MUST be SHA-2 with 256-bits of output
(`0x12`).
```

... and replace it with an example base64url value without padding.